blog/docs/index.md

```diff
@@ -32,10 +32,10 @@ Details on the Steering Committee 2021 election results.
 
 [Read more :octicons-arrow-right-24:](steering/2021-12-14-steering-elections-results.md){ .md-button }
 
-## Knative 1.0 is out!
-Details on the 1.0 release of the Knative project.
+## Knative 1.2 is out!
```

Comment:
I'm not sure how we decide on the order of the blog featured posts. Should the 1.2 release post be above the CNCF and Steering Committee posts because it is newer?

cc: @csantanapr

Reply:
Cncf donation should be on top, then new release 1.2

Remove the blog “Welcome new members to Steering Committee”

Reply:
Done